Cargo.toml

```diff
@@ -14,7 +14,7 @@ include = ["src/**/*", "LICENSE", "README.md"]
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3"  }
 futures = "0.3"
-subxt = "0.28"
+subxt = { version = "0.32.1", features = ["substrate-compat"] }
```

Comment:
Looks good but there's an even newer version now: https://crates.io/crates/subxt/0.33.0

Reply:
thanks for review @ascjones, please find it now updated to 0.33